rest/replication_api_test.go

```diff
@@ -1521,6 +1521,40 @@ func TestSGR1CheckpointMigrationPush(t *testing.T) {
 	assert.Equal(t, int64(1), r.Push.Checkpointer.Stats().GetCheckpointMissCount)
 }
 
+func TestReplicationConfig(t *testing.T) {
```

Comment:
Might be worth just adding extra test cases in `TestValidateReplication` instead of a new test?

Reply:
Yup, fair, not sure how I missed that.